cirq/ops/common_gates_test.py

```diff
@@ -530,6 +530,78 @@ def test_rz_unitary():
                                np.array([[1j, 0], [0, -1j]]))
 
 
+def test_x_stabilizer():
+    gate = cirq.X
+    assert cirq.has_stabilizer_effect(gate)
+    assert not cirq.has_stabilizer_effect(gate**0.5)
+    assert cirq.has_stabilizer_effect(gate**0)
+    assert not cirq.has_stabilizer_effect(gate**-0.5)
+    assert cirq.has_stabilizer_effect(gate**4)
+    assert not cirq.has_stabilizer_effect(gate**1.2)
+    foo = sympy.Symbol('foo')
+    assert not cirq.has_stabilizer_effect(gate**foo)
+
+
+def test_y_stabilizer():
+    gate = cirq.Y
+    assert cirq.has_stabilizer_effect(gate)
+    assert not cirq.has_stabilizer_effect(gate**0.5)
```

Comment:
Square root of Y is a stabilizer gate.

Reply:
As I mentioned in https://github.com/quantumlib/Cirq/pull/2760#issuecomment-585366509, I'm just adding support for the operations that CliffordState supports right now (just like it was before this) and does not try to decompose. Y**0.5 is a stabilizer gate and so is iSwap but just restricting myself to the original 7 gates before #2803 adds all of them to CliffordState since right now the simulator would just balk at Y**0.5.

If you feel we should add all these gates to the protocol at once, I can wait for the #2803 to go in. I can then have the protocol just do what is_supported_gate doing now as described in https://github.com/quantumlib/Cirq/pull/2803/files#r390152850. It would save on double work and definitely seems like a better option than having to define _has_stabilizer_effect_ on all gates.